### PR TITLE
[GPU] Support implicit crop in simple format.

### PR DESCRIPTION
**Pull request by hyunback**
_For Stable Diffusion Unet performance improvement,_
- _Support implicit crop for all axis in simple format_
- _Enable Eltwise fusion in fs_byx_fsv32 format for special pattern.(Actually Eltwise run simple format)_

### Tickets:
 - *137950*

### CODE REVIEW

**Review by e-ddykim — APPROVED** [review]
Looks good to me

File: src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp [hunk 1]
```diff
@@ -564,11 +564,25 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
         };
 
         auto eltwise_supports_fusings = [&](eltwise_node& node) -> bool {
+            auto has_reorder_hehind_mvn = [&]() -> bool {
```
Comment (yeonbok) [inline src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp:567]: minor: hehind => behind

**Review by yeonbok — APPROVED** [review]

**Comment by yeonbok** [comment]
LGTM except the typo :)

File: src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp (hunk 1 above)
Comment (hyunback) [reply src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp]: Oh, updated!

**Pull request merged**